src/components/UserInfo/UserInfo.tsx

```diff
@@ -1,5 +1,17 @@
 import React from 'react';
+import { User } from '../../types/User';
+import './UserInfo.scss';
 
-export const UserInfo: React.FC = () => (
-  <>Put the user here</>
+type Props = {
+  user: User;
+};
+
+export const UserInfo: React.FC<Props> = ({ user }) => (
```

Comment:
Use array destructuring to destructure the `user` prop. For example: `export const UserInfo: React.FC<Props> = ({ user: { name, email } }) => (`